apps/modernization-ui/src/apps/page-builder/components/AlertBase/AlertBase.tsx

```diff
@@ -0,0 +1,29 @@
+import './AlertBase.scss';
+import { Icon } from '@trussworks/react-uswds';
+import React from 'react';
+
+export type AlertBannerProps = {
+    type?: string;
+    children?: any;
+    message?: string;
+    header?: string;
+};
+
+export const AlertBase = ({ type, message, header }: AlertBannerProps) => {
```

Comment:
Issue: props are named `AlertBannerProps` but component is `AlertBase`. These typically match in name.